capstone_project/src/main/java/com/google/sps/servlets/PostServlet.java

```diff
@@ -34,6 +36,12 @@
 public class PostServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    if (!Authenticator.isLoggedIn(response, "/pages/comments.jsp")) {
+      return;
+    }
+
+    Resource resource = new Resource();
```

Comment:
Does resource get used anywhere?

Reply:
it'll be used in my next pr. as of right now it adds resources to datastore